### ISSUE TITLE
Ready to buy hardware?

### ISSUE THREAD

**Issue by RoMaTiX99** [Mat931/esp32-doorbell-bus-interface#10]
Hi there! 

I wonder whether anyone has a working pcb laying around at home and is willing to sell it? 
I'm not familiar with pcb creation - however I managed to process it on JLCPCB (thanks for the good description!). 

For me the price point of roughly 70$ is quite high - especially since I don't need 5 pieces. 

So if anybody has some left-overs and is willing to sell them (shipping to Germany) I'd be really grateful! 

Best regards!

**Comment by bladecell** [comment]
Hey, I have just received 5 of them so I could spare one or two.. send me an email if you are interested. 
estate-woozy-broom@duck.com

**Comment by RoMaTiX99** [comment]
Hi @bladecell ,

thank you - we managed to organize a group buy in the other open issue. So I don’t have any need currently. 

However I will leave this issue open so others can see your offer. 

Best!

**Comment by BreBar07** [comment]
Hey all,

Just received my 5 complete boards, I only need one.

Anyone who needs one yell out happy to sell at cost + shipping, I’m based in Australia.

Thanks!

**Comment by mikkelke** [comment]
Hi! 

I have four complete boards that i do not need. 
If anyone in Europe need i will sell for the cost + shipping. I'm from Denmark.

**Comment by Poldo2005** [comment]
Hi @mikkelke , I'm interested in buying a complete board (or two). Could you send me the price, including shipping costs to Italy, to the address indicated in the image? 

<img width="126" alt="Image" src="https://github.com/user-attachments/assets/fbe43253-0532-4172-aa5c-6e2015997b55" />

**Comment by zeratul021** [comment]
I would also join the fun if anyone is ordering in bulk to EU (I am in Czechia). Hit me up with pricing details, much appreciated:
range.view6078@eagereverest.com

**Comment by Chillidaddy** [comment]
> Hi!
> 
> I have four complete boards that i do not need. If anyone in Europe need i will sell for the cost + shipping. I'm from Denmark.

Hello,
from Germany. If you have a board left I’m interested. 

popeline-sprueche-0u@icloud.com
(throwaway iCloud Email address.)

**Comment by klausellus-wallace** [comment]
I'd like to join the ride too. HMU if you got spare boards flying around, for shipping in EU.![image](https://github.com/user-attachments/assets/1dd76736-a404-43ec-a1f6-e0011eebcc62)

**Comment by harl-e** [comment]
Hello, I am looking for a board at an affordable price with shipping to Germany.
Alternatively, I am also interested in people who want to share the cost of a new order.

**Comment by JTorkk** [comment]
> Hello, I am looking for a board at an affordable price with shipping to Germany. Alternatively, I am also interested in people who want to share the cost of a new order.

I have few left and can send to Germany. Send me a email at: [juho@torkke.li](mailto:juho@torkke.li)

**Comment by coldhand7** [comment]
Hello @rafuz, sorry. Just yesterday I've got a board from a nice swiss guy.

**Comment by sagitt** [comment]
hi, anyone have a board to sell to me? thanks
to italy

**Comment by chochb** [comment]
I found one on eBay!

Hopefully it works too :)

Does anyone have a 3D printer template for a suitable case?

**Comment by sagitt** [comment]
> I found one on eBay!
> 
> Hopefully it works too :)
> 
> Does anyone have a 3D printer template for a suitable case?

https://github.com/Mat931/esp32-doorbell-bus-interface/issues/26

Here

**Comment by zwinsch64** [comment]
first: so many thanks to Mat931 for sharing his great project! Awesome!
(I would love to have also Audio and Video... but this seems to be quiet a task, following what can be found in the web)

at adamjez et all:
I did order 5 Boards from JLCPCB on Dec. 30th and they did arrive today. 
I am willing to give away 3 to 4 of them. (lets say: 20€ plus shipping costs - from Frankfurt a/M, Germany)
The boards are fully assembled **but: WITHOUT the ESP** and without the Bus-Header!
I also have spare ESP Solo1 and Headers - so in case you really need it: I could additionally solder them (by hand) to the board.

**Comment by SetupCoding** [comment]
Edit 2026-04-17: I still have a couple of pre-flashed boards left. 
You can update them with your config over the air via Wi-Fi without needing an FT232RL (or any USB-to-serial adapter) anymore.

Edit 2026-02-17: I ordered a second batch of boards for my neighbors that were also very interested in automating their intercom and will have some spares left again. 

I am shipping inside EU (from Germany) as a tracked Parcel via DHL.

If you are interested I am willing to sell a board for 30€ + shipping.

Feel free to reach out to me: anton.setup.schmidt+abb@gmail.com

--- 
Initial comment:

I ordered 5 boards and was able to make everything work with the first one on my BUSCH-JAEGER 83205 AP-6 intercom.
Flashing was a learning curve though.

~~Thus I have 4 boards with female jumpers and an ESP32 module each for sale.~~
Edit 2026-02-16: All 4 boards from my initial order are sold.

<img width="400" height="auto" alt="Image" src="https://github.com/user-attachments/assets/85aac694-dca3-4e7a-9d57-6ad770f0ddb4" />

**Comment by lopez3** [comment]
I'd be interested in buying if anyone has a spare one, shipping to Poland. githubabb.repave347@passmail.net 
Thanks!

edit: a board is already on it's way to me, thank you for help :)

**Comment by asiersan** [comment]
I’m interested too.  Shipping to spain.

**Comment by zwinsch64** [comment]
Hallo Asiersan,
 
I still have 2 left, and could send you one. 

Tell me if you want to have it with ESP and/or Headers (soldered or loose)...
And send me via PPal friends 28 (without ESP and Headers) or 32 (With ESP and Headers soldered) to my PPal account: ***@***.*** ***@***.*** (attn: different from my email address)
And, of course, give me your postal address.
greez
Karsten

**Comment by asiersan** [comment]
With ESP and Headers soldered, i cant see your e-mail here.

**Comment by zwinsch64** [comment]
@asiersan 
> With ESP and Headers soldered, i cant see your e-mail here.
<img width="122" height="31" alt="Image" src="https://github.com/user-attachments/assets/77ffa750-848d-4ea8-89bb-f0ae97d00710" />

**Comment by asiersan** [comment]
> [@asiersan](https://github.com/asiersan)
> 
> > With ESP and Headers soldered, i cant see your e-mail here.
> 
> <img alt="Image" width="122" height="31" src="https://private-user-images.githubusercontent.com/166884729/579526284-77ffa750-848d-4ea8-89bb-f0ae97d00710.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NzY0MTA0NTYsIm5iZiI6MTc3NjQxMDE1NiwicGF0aCI6Ii8xNjY4ODQ3MjkvNTc5NTI2Mjg0LTc3ZmZhNzUwLTg0OGQtNGVhOC04OWJiLWYwYWU5N2QwMDcxMC5wbmc_lo WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjYwNDE3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI2MDQxN1QwNzE1NTZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wZDZhZjM5ZmQ5MDAzYzUxMzkzYzIxMzQ4NTBjM2E3MzU2NDVkOTk2YzNjNGFjZWMwMGI3MWZhNjQyY2IzZDMxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZyZXNwb25zZS1jb250ZW50LXR5cGU9aW1hZ2UlMkZwbmcifQ.-f9rN9XqJVIfue4WRax2t1CPBHH8QzPEfrZUD0AjBzk">
Done, give me a  mail for addres asiersan@gmail.com

**Comment by zwinsch64** [comment]
Hallo @asiersan,

I did not receive any, neither email with postal address, nor ppal payment bey now.
Sure you did use my address as shown above (karsten dot f dot w at gmx dot de)?

**Comment by javicalle** [comment]
Anyone interested in a group buy? I’m located in Spain and happy to ship within the EU.

I'm interested in the device and have started looking into how to order it. For now, it seems the ESP32s are unavailable. From the photo, it looks like I could get the other one they do have available (ESP32-SOLO-1(4MB)
C2934562), but comparing the PCB footprint diagram is confusing. Can anyone confirm if it's possible to swap them?

**Comment by SetupCoding** [comment]
Hey @javicalle I still have plenty devices (pre flashed for easy provisioning) with ESP32 modules on them and am shipping in Europe. If you want to buy one (30€ + tracked DHL shipping), feel free to contact me: anton.setup.schmidt+abb@gmail.com

**Comment by threadstone83** [comment]
@SetupCoding do you have another one left? If so, I'd like one too.

Or if anyone else have one lift: I'm interested :-)

**Comment by SetupCoding** [comment]
Hey @threadstone83 I still have plenty devices (pre flashed for easy provisioning) with ESP32 modules on them and am shipping in Europe. If you want to buy one (30€ + tracked DHL shipping), feel free to contact me with your shipping address: anton.setup.schmidt+abb@gmail.com

**Comment by bordelloweb** [comment]
Hi everyone,

I’m looking to get one of the pre-flashed ESP32 devices for ABB integration mentioned here, but shipping costs to Italy are a bit high for a single unit.

Is anyone in Europe interested in doing a group buy / shared shipping?
Or does anyone already have a spare device available for sale?

Feel free to reply here or DM me.

Thanks!

**Comment by SetupCoding** [comment]
Hey @bordelloweb  I still have plenty devices (pre flashed for easy provisioning) with ESP32 modules on them and am shipping in Europe. If you want to buy one (30€ + tracked DHL shipping), feel free to contact me with your shipping address: anton.setup.schmidt+abb@gmail.com

Shipping to Italy via DHL (tracking + insurance is 14.49€)
So a total of 44.49€ for one board. I can send it out on Friday.

**Comment by sagitt** [comment]
> Hey @bordelloweb  I still have plenty devices (pre flashed for easy provisioning) with ESP32 modules on them and am shipping in Europe. If you want to buy one (30€ + tracked DHL shipping), feel free to contact me with your shipping address: anton.setup.schmidt+abb@gmail.com
> 
> Shipping to Italy via DHL (tracking + insurance is 14.49€)
> So a total of 44.49€ for one board. I can send it out on Friday. 
> 

If it’s the one soldered by the manufacturer, I’m interested.

**Comment by SetupCoding** [comment]
@sagitt Yes it is. Feel free to contact me with your shipping address: anton.setup.schmidt+abb@gmail.com
i will ship it to you on Friday.

**Comment by sagitt** [comment]
> @sagitt Yes it is. Feel free to contact me with your shipping address: anton.setup.schmidt+abb@gmail.com
> i will ship it to you on Friday. 

I sent a mail to you.

**Comment by buddhaCode** [comment]
@SetupCoding Do you still have one of these boards? I send you a mail a couple of days ago.

**Comment by SetupCoding** [comment]
Hey @buddhaCode, 

unfortunately I did not receive any e-mail. Are you sure you wrote to anton.setup.schmidt+abb@gmail.com ?
I still have plenty of boards and am able to ship them out on the 20th of June.

**Comment by buddhaCode** [comment]
@SetupCoding Alright. Did a second try. Can u maybe check your spam folder?

1st try on June, 1st 8:23 pm CEST
2nd right now, June, 11th 3:29 pm CEST
from a***.b*****@mailbox.org

**Comment by VVlasy** [comment]
I will have boards for sale. Will include an eBay listing, although from me, fully assembled and pre-flashed the cost will be 99€ so not sure about the interest. I had a full order of 5 boards come to 350€

https://www.ebay.de/itm/287464105985

**Comment by SetupCoding** [comment]
@VVlasy Yeah I also still have Boards left over (assembled and pre-flashed) selling them for 30€ + shipping in EU.

@buddhaCode @threadstone83 and a couple of others from this thread have already ordered and are using them already :) 

Additionally: I always wanted to have a write up of my journey with automating the doorbell, so here goes, if anyone is interested: https://setup-coding.de/blog/busch-jaeger-home-assistant

**Comment by Sschhsd** [comment]
Hey @SetupCoding , I am also interested.
Did you receive my mail?

**Comment by asiersan** [comment]
I have a board in Spain if anybody is interested. I tried it it works, but an ha integración brings me same and more in a wifi monitor

**Comment by i-snake-z** [comment]
> I have a board in Spain if anybody is interested. I tried it it works, but an ha integración brings me same and more in a wifi monitor

How much for the board. I am from Portugal. Did you manage to connect it ? It’s the board without the white noise ?

**Comment by asiersan** [comment]
> > I have a board in Spain if anybody is interested. I tried it it works, but an ha integración brings me same and more in a wifi monitor
> 
> How much for the board. I am from Portugal. Did you manage to connect it ? It’s the board without the white noise ?
> 

- Is the original board. Is preflashed the cost is 30€ the same that cost me+ postal to Portugal.
I send you the code i used by mail if you whant add to esphome manually.
Contact me at asiersan@gmail.com

**Comment by fquinto** [comment]
Hi everyone,

I’ve designed a new, updated version of the electronic board for this project. The design is ready, but **the board has not been manufactured or tested yet**, so it should be considered an **untested prototype**.

I’m looking for people who might be interested in joining a **group buy / small production run** to have the boards manufactured and then test them together.

Project: https://github.com/fquinto/abb-welcome-gateway
Contact / discussion: https://t.me/abb_welcome_gateway

If you’re interested in joining the group buy or helping with testing, please let me know!
